From 4ea13a8ab03bb4c67e112a17baaec99f2bd3afcb Mon Sep 17 00:00:00 2001
From: svenate <salini.venate@intel.com>
Date: Wed, 16 Mar 2022 00:18:15 +0530
Subject: [PATCH] Replace libavservices_minijail_vendor with
 libavservices_minijail

Replace libavservices_minijail_vendor with
libavservices_minijail as per changes from aosp code.

Signed-off-by: svenate <salini.venate@intel.com>
---
 c2_store/Android.bp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/c2_store/Android.bp b/c2_store/Android.bp
index facf00f..2e71675 100644
--- a/c2_store/Android.bp
+++ b/c2_store/Android.bp
@@ -49,7 +49,7 @@ cc_binary {
 
     shared_libs: [
         "android.hardware.media.omx@1.0",
-        "libavservices_minijail_vendor",
+        "libavservices_minijail",
         "libbinder",
         "libhidltransport",
         "libhwbinder",
-- 
2.17.1

